File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseMantenimiento.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.MantenimientoDTO;

/**
 * Objeto de respuesta con la informacion del mantenimiento
 * 
 * @author Flor
 *
 */
public class ServerResponseMantenimiento {

	private String idMantenimiento = null;

	private List<MantenimientoDTO> listaMantenimiento = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseMantenimiento() {
		super();
	}

	public String getIdMantenimiento() {
		return idMantenimiento;
	}

	public void setIdMantenimiento(String idMantenimiento) {
		this.idMantenimiento = idMantenimiento;
	}

	public List<MantenimientoDTO> getListaMantenimiento() {
		return listaMantenimiento;
	}

	public void setListaMantenimiento(List<MantenimientoDTO> listaMantenimiento) {
		this.listaMantenimiento = listaMantenimiento;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseMantenimiento.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseMantenimiento
    {
        private string _idMantenimiento;

        public string idMantenimiento
        {
            get { return _idMantenimiento; }
            set { _idMantenimiento = value; }
        }

        private List<MantenimientoDTO> _listaMantenimiento;

        public List<MantenimientoDTO> listaMantenimiento
        {
            get { return _listaMantenimiento; }
            set { _listaMantenimiento = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
